src/Controller.cpp

```diff
@@ -669,7 +669,9 @@ SynchMempoolTask::~SynchMempoolTask() {
         if (!scriptHashesAffected.empty()) {
             // notify status change for affected sh's, regardless of how this task exited (this catches corner cases
             // where we queued up some notifications and then we died on a retry due to errors from bitcoind)
+            auto copy = std::unordered_set<HashX, HashHasher>(scriptHashesAffected);
```

Comment:
You can just do `auto copy = scriptHashesAffected;`

Reply:
Noted